docs/sponsors.rst

```diff
@@ -1,12 +1,49 @@
-Sponsors
-========
+Sponsors and Supporters
+=======================
 
 Please consider sponsoring urllib3 development, especially if your company
 benefits from this library.
 
 Your contribution will go towards adding new features to urllib3 and making
 sure all functionality continues to meet our high quality standards.
 
+
+v2.0 Sponsor Perks
+------------------
+
+.. important::
+
+   `Get in contact <mailto:sethmichaellarson@gmail.com>`_ for additional
+   details on sponsorship and perks before making a contribution
+   through `Open Collective <https://opencollective.com/urllib3>`_ if you have questions.
+
+
+Gold v2.0 Sponsors Perks
```

Comment:
On Open Collective and in the Silver perk below, there's no s after Sponsor. I don't have an opinion but it should be consistent in my humble opinion

Reply:
I have a todo item to fix this, thanks!